### PR TITLE
Add Shadow Mangas

### LINKED ISSUE #8203: ShadowMangas
### Acknowledgements

- [x] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [x] I have written a title with source name.
- [x] I have checked that the extension does not already exist on the [website extensions list](https://tachiyomi.org/extensions/) or the app.
- [x] I have checked that the extension does not already exist by searching the [GitHub repository](https://github.com/tachiyomiorg/tachiyomi-extensions/) and verified it does not appear in the code base.
- [x] I will fill out all of the requested information in this form.

### Source name

ShadowMangas

### Source link

https://shadowmangas.com/

### Language

Spanish

### Other details

Please Add this web with different genres of series.
Here is a preview of the web for the theme. 
Thanks in advance.

Screenshot_20210720_125135_com android chrome

### PR DESCRIPTION
**Pull request by DTecheira**
Closes #8203 #9007

Checklist:

- [x] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [x] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [x] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio

Screen Shot 2022-03-16 at 18 12 54
Screen Shot 2022-03-16 at 18 13 09

### CODE REVIEW

**Comment by DTecheira** [comment]
What would I be missing?

File: multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/wpmangastream/WPMangaStreamGenerator.kt [hunk 1]
```diff
@@ -54,6 +54,7 @@ class WPMangaStreamGenerator : ThemeSourceGenerator {
         SingleLang("Vapo Scan", "https://vaposcans.com", "pt-BR", overrideVersionCode = 3),
         SingleLang("West Manga", "https://westmanga.info", "id", overrideVersionCode = 1),
         SingleLang("xCaliBR Scans", "https://xcalibrscans.com", "en", overrideVersionCode = 3),
+        SingleLang("Shadow Mangas", "https://shadowmangas.com", "es", overrideVersionCode = 1),
```
Comment (alessandrojean) [inline multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/wpmangastream/WPMangaStreamGenerator.kt:57]: If it's the first version, you can omit the `overrideVersionCode`.

```suggestion
        SingleLang("Shadow Mangas", "https://shadowmangas.com", "es"),
```

**Review by alessandrojean — CHANGES_REQUESTED** [review]

**Comment by vetleledaal** [comment]
CI appears to have failed on something unrelated, you can push a new commit to retry

**Comment by arkon** [comment]
@mergifyio rebase

**Pull request merged**